tests/testthat/test_check_result_df.R

```diff
@@ -0,0 +1,112 @@
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x44th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x45th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x46th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x47th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x48th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x49th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x50th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x51st.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x52nd.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x53rd.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x54th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x55th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x56th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x57th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x58th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x59th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x60th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x61st.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x62nd.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x63rd.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x64th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x65th.week = c(NA_real_, 
+    NA_real_, NA_real_, NA_real_, NA_real_, NA_real_), x66th.week = c(NA, 
+    NA, NA, NA, NA, NA), x67th.week = c(NA, NA, NA, NA, NA, NA
+    ), x68th.week = c(NA, NA, NA, NA, NA, NA), x69th.week = c(NA, 
+    NA, NA, NA, NA, NA), x70th.week = c(NA, NA, NA, NA, NA, NA
+    ), x71st.week = c(NA, NA, NA, NA, NA, NA), x72nd.week = c(NA, 
+    NA, NA, NA, NA, NA), x73rd.week = c(NA, NA, NA, NA, NA, NA
+    ), x74th.week = c(NA, NA, NA, NA, NA, NA), x75th.week = c(NA, 
+    NA, NA, NA, NA, NA), x76th.week = c(NA, NA, NA, NA, NA, NA
+    )), class = c("tbl_df", "tbl", "data.frame"), row.names = c(NA, 
+-6L))
+# nolint end
+
+user <- reshape2::melt(
+  head(billboard),
+  id.vars = c("year", "artist.inverted", "track", "time",
+              "genre", "date.entered", "date.peaked"),
+  variable.name = "week",
+  value.name = "rank",
+  factorAsStrings = TRUE
+)
+user$week <- as.character(user$week)
+solution <- tidyr::gather(head(billboard), "week", "rank", x1st.week:x76th.week)
+
+user <- tibble::as_tibble(user)
+solution <- tibble::as_tibble(solution)
+
+test_that("Comparing dataframes, testing for null env", {
```

Comment:
need to insert a `skip_if_not_installed("tidyr")`. Should be done with all suggested packages